va/validation-authority.go

```diff
@@ -36,12 +36,13 @@ var ErrTooManyCNAME = errors.New("too many CNAME/DNAME lookups")
 
 // ValidationAuthorityImpl represents a VA
 type ValidationAuthorityImpl struct {
-	RA           core.RegistrationAuthority
-	log          *blog.AuditLogger
-	DNSResolver  core.DNSResolver
-	IssuerDomain string
-	TestMode     bool
-	UserAgent    string
+	RA            core.RegistrationAuthority
+	log           *blog.AuditLogger
+	DNSResolver   core.DNSResolver
+	IssuerDomain  string
+	TestMode      bool
+	UserAgent     string
+	AddressFilter int
```

Comment:
Okay, offline feedback happened. 

Basically, we want to support ipv6 IP addresses in the DNS resolution and our current infra can't handle it. Roland is simplifying this to be just a IPv4-only filter vs a both filter.

I'm not super stoked about writing code for stuff we won't be using in production, but okay.